website/src/main/resources/public/index.html

```diff
@@ -92,7 +92,7 @@ <h1><span class="image right"><img style="width: 50%" src="images/logo_frame.png
     <!-- One -->
     <section id="one" class="wrapper style2 spotlights">
         <section>
-        <span class="image"><img src="images/questions.png" alt="Questions and Answers"
```

Comment:
please keep the old image name and replace it instead of adding a new image, thanks